roles/create_rds_global_cluster/tasks/create.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Run 'create_rds_global_cluster' role create operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Create rds global database
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"
+        engine: "{{ create_rds_global_cluster_enigne  | default('aurora-postgresql')}}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        state: present
+
+    - name: Create a primary cluster for global database "{{ create_rds_global_cluster_global_cluster_name }}" in "{{ create_rds_global_cluster_primary_cluster_region }}"
+      amazon.aws.rds_cluster:
+        db_cluster_identifier: "{{ create_rds_global_cluster_primary_cluster_name | default('{{ create_rds_global_cluster_global_cluster_name }}') }}" #aka cluster_name
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        engine: "{{ create_rds_global_cluster_enigne }}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
+        username: "{{ create_rds_global_cluster_username }}"
+        password: "{{ create_rds_global_cluster_password }}"
+        db_subnet_group_name: "{{ create_rds_global_cluster_db_subnet_group_name | default(omit) }}"
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"  # what global db cluster should be associated to
+      ignore_errors: true
+
+    - name: Create an instance connected to primary cluster
+      amazon.aws.rds_instance:
+        db_cluster_identifier: "{{ create_rds_global_cluster_primary_cluster_name }}" # what cluster instance should be in
+        db_instance_identifier: "{{ create_rds_global_cluster_primary_cluster_instance_name }}"
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        engine: "{{ create_rds_global_cluster_enigne }}"
+        db_instance_class: "{{ create_rds_global_cluster_instance_class }}"
+
+    - name: Create a read replica cluster for global database "{{ create_rds_global_cluster_global_cluster_name }}" in "{{ create_rds_global_cluster_replica_cluster_region }}"
```

Comment:
Can we let the user decide if they want a read or write replica?

Reply:
thanks, will add this option

Reply:
@GomathiselviS  it looks like, the secondary clusters can only be `read` replicas and not `write`

```
Secondary Regions can serve low latency reads. In the unlikely event that your database becomes degraded or isolated in the primary Region, you can promote your secondary Region.
```